checklists/alz_checklist.en.json

```diff
@@ -1763,28 +1708,6 @@
             "id": "E01.13",
             "severity": "Medium"
         },
-        {
-            "category": "Governance",
-            "subcategory": "Optimize your cloud investment",
-            "text": "Consider using automation tags to start/stop VM's in your environment to save on cost.",
-            "waf": "Security",
-            "service": "VM",
-            "guid": "9b5e2a28-9823-4faf-ab7e-afa5f6c57221",
-            "id": "E02.01",
-            "severity": "Low",
-            "link": "https://learn.microsoft.com/azure/automation/automation-solution-vm-management-config"
-        },
-        {
-            "category": "Management",
-            "subcategory": "Scalability",
-            "text": "Leverage Azure Virtual Machine Scale sets to scale in and out based on the load.",
```

Comment:
Agree with this being removed, but should we store it away for use in a Scale set checklist?